spec/shopify_transporter/pipeline/magento/product/top_level_attributes_spec.rb

```diff
@@ -5,6 +5,13 @@
 module ShopifyTransporter::Pipeline::Magento::Product
   RSpec.describe TopLevelAttributes, type: :helper do
     context '#convert' do
+      it 'does not accumulate anything if product is not a top level product' do
+        magento_product = FactoryBot.build(:magento_product, :with_parent_id)
+        shopify_product = described_class.new.convert(magento_product, {})
+
+        expect(shopify_product).to eq({})
```

Comment:
Because TopLevelAttributes mutates the input, I suggest:
```
shopify_product = {}
described_class.new.convert(magento_product, shopify_product)
expect(shopify_product).to eq({})
```